docs/source/cli_help/anno.rst

```diff
@@ -21,7 +21,9 @@ If invoked with a single empty string <annotation text>, like:
 then clears the annotation for the current branch (or a branch specified with ``-b/--branch``).
 
 If invoked with ``-H`` or ``--sync-github-prs``, annotates the branches based on their corresponding GitHub PR numbers and authors.
-Any existing annotations are overwritten for the branches that have an opened PR; annotations for the other branches remain untouched.
+When the current user is **not** the owner of the PR associated with that branch, adds ``rebase=no push=no`` branch qualifiers used by ``git machete traverse``,
+so that you don't rebase or push someone else's PR by accident.
+Any existing annotations (except branch qualifiers) are overwritten for the branches that have an opened PR; annotations for the other branches remain untouched.
```

Comment:
For the branches corresponding to the **user's own** PRs... the qualifiers are **not** overwritten, right?

Reply:
You are right, they are not overwritten

Reply:
Okay, let's leave as is, isn't as important to elaborate